airflow_dags/wikipedia/word_frequency/dolt_load.py

```diff
@@ -19,7 +20,7 @@
 CURR_DIR = path.dirname(path.abspath(__file__))
 BZ2_FILE_NAME = 'enwiki-latest-pages-articles-multistream.xml.bz2'
 DUMP_URL = 'https://dumps.wikimedia.your.org/enwiki/latest/{}'.format(BZ2_FILE_NAME)
-WIKIEXTRACTOR_PATH = path.join(CURR_DIR, 'wikiextractor/WikiExtractor.py')
+WIKIEXTRACTOR_PATH = path.join(Path(CURR_DIR).parent, 'wikiextractor/WikiExtractor.py')
```

Comment:
If possible we should use `pip` here. It can be added as a dependency for the project, though @reltuk would have to comment on how this would effect our deployment procedure (i.e. it might be the case that the job fails until manual intervention to force the machine to restart and install dependencies fresh from pip).

Reply:
Wikiextractor won't work with pip in its current state, so going to make an issue for this and update it at a later point. But I agree we should make this change at some point.